### DISCUSSION TITLE
[A-L] (2023/24) Foglio 3 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#147] [Esercizi - Bertini] [answered]
![Es 1](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/e09057a4-892b-4f38-8eb5-e4ae5bab9224)

**Comment by Elia-Belli** [comment]
**Nota**: siccome nel **(3)(4)(5)** si parla di partizioni, considero i gruppi indistinguibili
- **(1)** Lo spazio degli eventi $\Omega$ è l'insieme delle permutazioni dei compiti (3 tipi distinti, 3 di ogni tipo), analogo alle permutazioni di $AAABBBCCC$: $|\Omega|=\frac{9!}{3!3!3!}$ 
- **(2)** Lo spazio degli eventi è sempre la permutazione dei compiti (tutti distinti, 1 di ogni tipo), analogo alle permutazioni di $ABCDEFGHI$: $|\Omega|=9!$
- **(3)** Numero gli studenti da $1$ a $9$, siccome i gruppi non sono distinti, ovvero $[(1,2,3)(4,5,6)(7,8,9)] = [(4,5,6)(1,2,3)(7,8,9)]$, eseguo 3 estrazioni non ordinate e divido per le possibili permutazioni dei 3 gruppi : $\binom{9}{3}\binom{6}{3}\binom{3}{3}/3!$
- **(4)** Analogamento al **(5)**, eseguo le 3 estrazioni e divido per le permutazioni, solo che stavolta il gruppo da 5 è distinto, quindi si divide solo per le permutazioni di quelli che hanno lo stesso numero di elementi: $\binom{9}{5}\binom{4}{2}\binom{2}{2}/2!$
- **(5)** Non vi sono più gruppi distinti quindi abbiamo solo $1$ modo di partizionare gli studenti, o se ragioniamo con la logica precedente $\binom{9}{1}\binom{8}{1}\binom{7}{1}\binom{6}{1}\binom{5}{1}\binom{4}{1}\binom{3}{1}\binom{2}{1}\binom{1}{1}/9!=9!/9!=1$

**Comment by LeoCosta02** [comment]
@Elia-Belli nel punto 1, essendo indistinguibili i gruppi, non bisogna fare il calcolo con le combinazioni di ripetizioni?
Considero AAB|ABB|CCC uguale a BAA|BAB|CCC

**Comment by Elia-Belli** [comment]
> @Elia-Belli nel punto 1, essendo indistinguibili i gruppi, non bisogna fare il calcolo con le combinazioni di ripetizioni? Considero AAB|ABB|CCC uguale a BAA|BAB|CCC

Nel punto 1 non abbiamo gruppi, inoltre dividendo per il fattoriale delle occorrenze sto eliminando i casi simili a quello che hai appena mostrato. C'è anche un'altra soluzione analoga, ovvero usare il binomiale $\binom{9}{3}\binom{6}{3}\binom{3}{3}$, che se svolgi torna uguale alla mia soluzione

**Comment by LeoCosta02** [comment]
@Elia-Belli  Quindi nel punto 1 abbiamo n=9 che è il numero di studenti e k=9 come numero di compiti totali a 3 a 3 diversi

**Comment by CiottoloMaggico** [comment, marked as answer]
# Soluzione confermata in classe
Analogo @Elia-Belli 
@LeoCosta02 usare il metodo "distribuzione elementi tra separatori" risulterebbe più complicato, forse impossibile, perchè il professore chiede che esattamente 3 studenti vengano assegnati ad ogni compito e soprattutto ogni studente è distinto.
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/fda122ad-f940-4771-ae4b-12072c830310)
